orchestration/dagster_orchestration/hca_manage/manage.py

```diff
@@ -191,6 +200,32 @@ def submit_soft_delete(self, target_table: str, target_path: str) -> str:
 
         return response.json()["id"]
 
+    def submit_snapshot_request(self, optional_qualifier: Optional[str] = None) -> str:
+        date_stamp = str(datetime.today().date()).replace("-", "")
+        if optional_qualifier:
+            # prepend an underscore if this string is present
+            optional_qualifier = f"_{optional_qualifier}"
+        snapshot_name = f"{self.dataset}___{date_stamp}{optional_qualifier}"
```

Comment:
⛏️ It'd be helpful to log this out as it's useful when debugging with Jade if something goes awry.